lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -263,9 +261,7 @@ def hispanic_latino(ethnic_group: str) -> list:
 
     ethnic_map = {
         "Not Hispanic or Latino": "no",
-        "Hispanic or Latino": "yes",
-        "Unknown": None,
-        "NULL": None
+        "Hispanic or Latino": "yes"
```

Comment:
For the future, leaving a trailing comma on the last element of a list or dictionary is good practice in Python, as it results in smaller diffs.

Reply:
I see that now, too.